src/types/PackageInfo.ts

```diff
@@ -5,10 +5,10 @@ export interface PackageInfo {
   dependencies?: { [dep: string]: string };
   devDependencies?: { [dep: string]: string };
   peerDependencies?: { [dep: string]: string };
-  private: boolean;
+  private?: boolean;
   group?: string;
-  pipeline: { [dep: string]: string[] };
-  scripts: { [dep: string]: string };
+  pipeline?: { [dep: string]: string[] };
+  scripts?: { [dep: string]: string };
```

Comment:
I made these optional to match Rush's definition of a package.json.

Reply:
okay! pipeline is not necessary here, so you might as well delete it (I could do that in a future PR as well)